spec/factories/project_steps.rb

```diff
@@ -0,0 +1,49 @@
+FactoryGirl.define do
+  factory :project_step do
+    # project_table 'Loan'
+    # project_id { create(:loan).id }
+    project { create(:loan) }
+    person
+    # need to make sure parent saved before assigning these
+    # summary { Faker::Lorem.sentence }
+    # details { Faker::Lorem.paragraph }
+    scheduled_date { Faker::Date.between(Date.civil(2004, 01, 01), Date.today) }
+    completed_date { Faker::Date.between(Date.civil(2004, 01, 01), Date.today) }
+    is_finalized [true,false].sample  ##JE todo: is ther a better pattern to assign a boolean?
```

Comment:
I think this is pretty much the best way to randomly assign booleans, though as a silly nit-picky style thing I think there should be a space after the comma :stuck_out_tongue_closed_eyes: